plugins/nf-validation/src/testResources/samplesheet_no_required.csv

```diff
@@ -1,3 +1,3 @@
 fastq_1,fastq_2,strandedness
 test1_fastq1.fastq.gz,test1_fastq2.fastq.gz,forward
-test2_fastq1.fastq.gz,,forward
```

Comment:
Any reason for this change?

Reply:
To test multiple missing fields :wink:

Reply:
Any chance we can keep both? I think there is benefit in testing `<value>,<missing>,<value>`

Reply:
Yes of course! I misread your first comment :p, I deleted them by mistake probably